zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartitionAdminAccess.java

```diff
@@ -206,6 +212,51 @@ public ActorFuture<Void> banInstance(final long processInstanceKey) {
     return future;
   }
 
+  @Override
+  public ActorFuture<Void> configureFlowControl(final FlowControlCfg flowControlCfg) {
+    final ActorFuture<Void> future = concurrencyControl.createFuture();
+    concurrencyControl.run(
+        () -> {
+          try {
+            final FlowControl flowControl = adminControl.getLogStream().getFlowControl();
+            if (flowControlCfg.getAppend() != null && flowControlCfg.getAppend().isEnabled()) {
```

Comment:
:question: Why the `isEnabled` check here? That means I can't disable it via the actuator.